vector/src/main/java/im/vector/riotx/features/home/room/detail/RoomDetailFragment.kt

```diff
@@ -977,7 +978,9 @@ class RoomDetailFragment @Inject constructor(
 
     @SuppressLint("SetTextI18n")
     override fun onMemberNameClicked(informationData: MessageInformationData) {
-        insertUserDisplayNameInTextEditor(informationData.memberName?.toString())
+        session.getUser(informationData.senderId)?.let {
```

Comment:
User does not contains the display name of this specific room. It's maybe a pb

Reply:
I have changed the PillImageSpan class to take directly displayName and avatar instead of a user. Like this can be used by the caller with User or RoomMember depending on context